### DISCUSSION TITLE
<title>New Developer</title>

### DISCUSSION THREAD

**Discussion by ArthurGomes03** [community/community#43412] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello people,

I'm Arthur and I'm new developer front-end, I've been studying front end for a while and I'm really enjoying this area, I've been putting some projects on my github if anyone has tips on my projects comment here, a big hug.

**Comment by doguedogue** [comment, marked as answer]
go ahead, very good projects 🚀

**Reply by ArthurGomes03** [reply]
Thanks !

**Reply by ArthurGomes03** [reply]
Thank you for your comment, do you have any tips for getting started?
